### DISCUSSION TITLE
adding sensor to Fancontrol?

### DISCUSSION THREAD

**Discussion by Bozothenutter** [Rem0o/FanControl.Releases#500] [Q&A]
Hi,
HW monitor can see my GPU memjunction temp.
is there a way of using this as a source to create a curve?

**Comment by boroborob** [comment]
Hi,

At the top right corner, click on the three dots, then Sensor settings and disable the NvApiWrapper and use LibreHardwareMonitor's GPU monitoring instead.
With that you should get GPU memjunction temp as well.
